android/java/org/chromium/chrome/browser/crypto_wallet/permission/BraveEthereumPermissionAccountsListAdapter.java

```diff
@@ -21,6 +21,7 @@
 import androidx.recyclerview.widget.RecyclerView;
```

Comment:
let's rename `BraveEthereumPermissionAccountsListAdapter.java` to something like `BraveDappsPermissionAccountsListAdapter.java` or just `BravePermissionAccountsListAdapter.java` as it's not related to only Ethereum anymore.

Reply:
fixed and updated the corresponding files Altogether:  https://github.com/brave/brave-core/pull/15094/commits/102b2be52d2dcfb3b31ebf3a5f70ee57f632300a